src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -123,6 +109,30 @@ public override ExportResult Export(in Batch<Activity> batch)
                     Console.WriteLine();
                 }
             }
+#if NET461 || NETSTANDARD2_0
+            else if (typeof(T) == typeof(LogRecord))
+            {
+                var rightPaddingLength = 30;
+                foreach (var item in batch)
+                {
+                    var logRecord = item as LogRecord;
+                    Console.WriteLine($"{"LogRecord.TraceId:".PadRight(rightPaddingLength)}{logRecord.TraceId}");
```

Comment:
nit: the output is a bit different from the Activity one (e.g. here we have the `LogRecord.Something`.
Might be nice to make them consistent. Not blocking though.

Reply:
It's LogRecord.propertyName right now, what you suggest it be? Should we use "Activity" instead of "LogRecord"?

Reply:
Would you help to update the example to use the console exporter? https://github.com/open-telemetry/opentelemetry-dotnet/blob/46094add40c8d4a674c95ea062f0b06dde115390/docs/logs/getting-started/Program.cs#L33

We can use it to drive the output format discussion.